src/main/java/com/gmail/goosius/siegewar/objects/Siege.java

```diff
@@ -86,30 +80,6 @@ public Siege(Town town) {
     public Town getTown() {
         return town;
     }
-
-	public long getScheduledEndTime() {
-        return scheduledEndTime;
-    }
-
-    public long getActualEndTime() {
-        return actualEndTime;
-    }
-
-    public void setStartTime(long startTime) {
-        this.startTime = startTime;
-    }
-
-    public void setScheduledEndTime(long scheduledEndTime) {
-        this.scheduledEndTime = scheduledEndTime;
-    }
-
-    public void setActualEndTime(long actualEndTime) {
-        this.actualEndTime = actualEndTime;
-    }
-
-    public long getStartTime() {
-        return startTime;
-    }
```

Comment:
These probably need to remain in a deprecated state, returning the current system time in millis when it is a long method.

Reply:
- As I understand it, @depreciated usually means that a method is still usable, but unsafe, and might stop working in future.
- However, that is not the case with these methods. Rather we know for sure that after this PR, their effects and returns will no longer work at all.
- Therefore how about we simply delete?. 
  - Because although it would force those few modder who used these methods, to fix their code, at least they wouldn't feel misled into thinking that there was a chance that the methods would work.

Reply:
Deleting it will throw a NoSuchMethodError in the plugin that was using this. It is usually kinder to let them fail more gracefully. It's up to you.

Reply:
Ok, yea I'll nuke them so.  If anyone complains they can send me a strongly worded letter.